src/test/java/org/apache/commons/configuration2/TestDataConfiguration.java

```diff
@@ -358,7 +347,7 @@ public void setUp() throws Exception {
         // email address
         conf.addProperty("email.string", "ebourg@apache.org");
         conf.addProperty("email.string.interpolated", "${email.string}");
-        conf.addProperty("email.object", createInternetAddress("ebourg@apache.org"));
+        conf.addProperty("email.object", new InternetAddress("ebourg@apache.org"));
```

Comment:
I don't think we want any PII here.

Reply:
I think this email was already in the code before this change. There might be more PII in other places, I'm +1 to removing it. Maybe we could do it in a follow up issue if that's easier.